### PR TITLE
WIP Mandyoc Documentation Added

### PR DESCRIPTION
**Pull request by jamisonassuncao**
- this is a work in progress (WIP), don't merge yet!
- included necessary files for the documentation website
- github actions file added (Agustina's)
- changed .gitignore

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "MANDYOC"
+copyright = "2020, Jamison Assunção"
```
Comment (aguspesce) [inline docs/conf.py:21]: copyright = "2021, Mandyoc"

**Comment by aguspesce** [comment]
@jamisonassuncao: I think that equation (2.3) is wrong... Can you revise  it?

**Comment by aguspesce** [comment]
@jamisonassuncao, @rafaelmds, @victorsacek, 

I improve the documentation readme file to use it as the README file in the reposiotory. 
My idea is to remplase the actual readme

File: docs/files/theory.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+.. math:: 
+	:label: mass-conservation
+
+	u_{i,i}=0 
+
+.. math:: 
+	:label: momentum-conservation
+
+	\sigma_{ij,j}+g\rho_0(1-\alpha(T-T_0))=0
```
Comment (victorsacek) [inline docs/files/theory.rst:21]: Please, see the equation for \rho in the JOSS paper.

**Review by victorsacek — APPROVED** [review]

File: docs/files/theory.rst (hunk 2 above)
Comment (jamisonassuncao) [reply docs/files/theory.rst:21]: I found the expression \rho=\rho_{0}(1-\alpha(T-T_{0})), the same as on the line 21, but g was missing and "i" subscript.

**Pull request merged**